packages/block-library/src/comments-pagination/block.json

```diff
@@ -20,7 +20,7 @@
 		"align": true,
 		"reusable": false,
 		"html": false,
-		"inserter": false,
+		"inserter": true,
```

Comment:
We can remove this line since it’s the default value.

Reply:
Yep!